### ISSUE TITLE
Figure out what the difference/purpose of Train Step Medium vs Short

### ISSUE THREAD

**Issue by roddtalebi** [ezCGP/ezCGP#243]
...gotta find it in the code first

**Comment by roddtalebi** [comment]
Ohhhh hold on!
I only just noticed that both the TrainingStep Medium AND Short are used in the block_arguments for the TrainConfig block...that means Anthony purposely wanted both to be available to the block's primitives to use...weird right? So let's go to the ONLY primitive used in that block and see if/how those arguement types are used...
https://github.com/ezCGP/ezCGP/blob/feature/simgan-support/codes/block_definitions/utilities/operators_simgan_train_config.py#L25

These positional arguments to the primitive
```
                        train_steps,
                        r_pretrain_steps,
                        d_pretrain_steps,
                        d_updates_per_train_step,
                        r_updates_per_train_step,
                        r_lr,
                        d_lr,
                        delta,
                        use_data_history
```
correspond to these argument types as defined in the operator_dict for that primitive
```
argument_types.ArgumentType_TrainingStepsMedium,
             argument_types.ArgumentType_TrainingStepsShort,
             argument_types.ArgumentType_TrainingStepsShort,
             argument_types.ArgumentType_Int1to5,
             argument_types.ArgumentType_Int1to5,
             argument_types.ArgumentType_LearningRate,
             argument_types.ArgumentType_LearningRate,
             argument_types.ArgumentType_LearningRate,
             argument_types.ArgumentType_Bool
```
So Medium is for regular training, and Short is for pretraining!

Interesting...so I guess keep both and just fill each ```choices``` attribute with values that make sense based off what you guys have been doing in your own simgan repo.

Make sense? (sorry, this is where the code get's all over the place a bit)

**Comment by anishthite** [comment]
Yeah, maybe we should rename them to be `PretrainingSteps` and `TrainingSteps` then. For the simgan repo we've been running with the follwoing

```
train-config:
  epochs: 10
  r-pretrain-steps: 500
  d-pretrain-steps: 400
  train-steps: 50000
  batch-size: 128
  r-updates-per-step: 2
  r-lr: 0.001
  delta: 0.0001
  d-lr: 0.0001
  d-updates-per-step: 1
```
I guess maybe have two separate TrainingSteps for discriminator and refiner? We've never really played with making the two a lot different so idk how much that helps

**Comment by roddtalebi** [comment]
- Ya I like that idea to rename them.
- I can add those hyperparams to the seed to make sure they match
- it'll be super easy to change the code to make one for refiner and one for discrim but for now let's keep it as-is i say

I'll take a look at commit history rn to see if you've jumped ahead to make the changes, if not I'll do it, push to simgan-support and close the issue

**Comment by roddtalebi** [comment]
I changed 50,000 to 5,000 ...would've been insane otherwise

**Comment by anishthite** [comment]
I haven't done it yet, I'm working on getting general time series to plot for the transform dataset

**Comment by roddtalebi** [comment]
crap. great start to the morning...forgot to switch branches

and for the record, I'm on west coast so it is the morning here so that's still an excuse im allowed to use!
